packages/action-group/test/action-group.test.ts

```diff
@@ -304,6 +308,17 @@ describe('ActionGroup', () => {
         const closed = oneEvent(el.children[3] as ActionMenu, 'sp-closed');
         await closed;
 
+        if (!isWebKit) {
+            sendMouse({
+                steps: [
+                    {
+                        position: [110, 110],
```

Comment:
Instead of hardcoding the x and y co-ordinate, can we get this by `getBoundingClientRect`